src/main/java/com/github/fishio/control/SinglePlayerController.java

```diff
@@ -27,9 +28,38 @@
  * The controller class of the single player game.
  */
 public class SinglePlayerController implements ScreenController {
```

Comment:
*Auto-generated comment*

    
    The class 'SinglePlayerController' has a Cyclomatic Complexity of 2 (Highest = 11).